packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -53,6 +53,7 @@ export const SECRETS_MANAGER_PARSE_OWNED_SECRET_NAME = '@aws-cdk/aws-secretsmana
 export const KMS_DEFAULT_KEY_POLICIES = '@aws-cdk/aws-kms:defaultKeyPolicies';
 export const S3_GRANT_WRITE_WITHOUT_ACL = '@aws-cdk/aws-s3:grantWriteWithoutAcl';
 export const ECS_REMOVE_DEFAULT_DESIRED_COUNT = '@aws-cdk/aws-ecs-patterns:removeDefaultDesiredCount';
+export const ECS_ADD_SECURITY_GROUP = '@aws-cdk/aws-ecs-patterns:addSecurityGroup';
```

Comment:
Let's call this something more descriptive. See below.

Reply:
changed to  ECS_ADD_SECURITY_GROPU_TO_ASG_CAPACITY_PROVIDERS